### ISSUE TITLE
unixFmt escape characters

### ISSUE THREAD

**Issue by jayasurian123** [spencermountain/spacetime#207]
unixFmt method is not formatting correctly with 'T' 
eg. from the unit test case
```js
let epoch = 1510850065194
let s = spacetime(epoch, 'Canada/Eastern')
s.unixFmt("yyyy-MM-dd'T'HH:mm:ssZ") // ISO8601Format
// output 2017-11-ddT11:34:25-0500
// desired output 2017-11-16T11:34:25-0500
```

Its somehow adds T to the previous character and makes it `ddT`.

I have tried to check it and fix my own but there were not enough test example case for the quoted string.

**Comment by spencermountain** [comment]
hey Jayasurian, thanks. Is there a reason why you add the apostrophes around the T?

it should work I think, like this:
```js
let epoch = 1510850065194
let s = spacetime(epoch, 'Canada/Eastern')
let str = s.unixFmt('yyyy-MM-ddTHH:mm:ssZ') // ISO8601Format
// 2017-11-16T11:34:25-0500
```
let me know if I've misunderstood
cheers

**Comment by KaylaBrady** [comment]
Hi -- having a similar issue here when trying to use unixFmt to format a time for display in French. 

```
s.unixFmt("HH'h'mm")
// output:  HHh49
// expected output: 14h49
```

**Comment by spencermountain** [comment]
ah - thanks Kayla. Good eye.
Will take a look at supporting the quote-style escaping. 
In the short-term, you'll probably have to split it into two fmt commands.
cheers

**Comment by KaylaBrady** [comment]
Thanks! It seems to be only an issue when there is not a space before the quoted character -- ex
```
s.unixFmt("HH 'h' mm")
// output: 14 h 49
// exected output: 14 h 49
```

**Comment by spencermountain** [comment]
fixed in `v6.12.2`
thanks!
